### DISCUSSION TITLE
Need clear steps to do auto instrumentation for windows, linux, mac

### DISCUSSION THREAD

**Discussion by ramanjaneyagupta** [open-telemetry/opentelemetry-dotnet-instrumentation#2075] [General] [answered]
I am new to the dotnet and trying to follow the instructions provided here 

https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation

for windows, it is so confusing and steps are not clear to me. and lets say i have very basic sample application it have the follwong lines code 

Console.WriteLine("Hello, World!");

var httpClient = new HttpClient();
var response = await httpClient.GetAsync("https://google.com/");
Console.WriteLine($"Response Status: {response.StatusCode}");

so if i want to add auto instrumentation could someone help with the proper steps that users(like me) can understand easily. 

I did for java and python and it is so clear for the auto instrumentation but for dotnet i am fidning hardtime to setup.

Help would be appreciated.

**Comment by pellared** [comment, marked as answer]
I created https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/2076
